ApplicationRequirements.txt

```diff
@@ -18,42 +33,107 @@
 	"New Password" Text Box
 	"Confirm Password" Text Box
 	"Create New Account" Button
```

Comment:
For the driver, we want to ask them to enter their car make, model, license plate, etc.
For the user we can ask them to enter their permanent shipping address so they can use that when they create new order requests instead of retyping their address over and over.

Reply:
So I initially had it so it would default to the user's address but then I thought "What if the user wants to send something from their address instead of deliver something to their address? What if the user is somewhere other than their default address? Etc... etc..." I think it'd be best if as an optional feature we can store an address book for a user and they can access that as a drop-down menu. For the base application that isn't necessary though.

Reply:
Oh that would be a really nice feature! the user would be able to manage their address book by CRUD operations

Reply:
I agree with zafer. The driver would need to enter thier car make model, and license plate. So users would have an idea of what car is coming to pick up or deliver the item.

Reply:
That’s smart, I can update that tomorrow, I’m busy today and dont have access to my computer (even this usb from my phone)

Reply:
Is*